app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -30,7 +30,11 @@ export const StoryItem = ({
 
   const childProps = {
     story,
-    onToggle: () => toggleStory(story.id, from),
+    onToggle: async () => {
+      if (true) {
```

Comment:
what's the reasoning behind this `if`?

`onToggle: async () => expandOrCollapseStory(story, from)`  should work

Reply:
It is a debug line... Sorry for that. I'll remove it.